ReactAndroid/src/main/java/com/facebook/react/modules/accessibilityinfo/AccessibilityInfoModule.java

```diff
@@ -189,6 +196,41 @@ public void announceForAccessibility(String message) {
     mAccessibilityManager.sendAccessibilityEvent(event);
   }
 
+  @Override
+  public void getInstalledAccessibilityServiceList(Callback successCallback) {
+    List<AccessibilityServiceInfo> servicesList =
+        mAccessibilityManager.getInstalledAccessibilityServiceList();
+
+     WritableArray result =
+        Arguments.fromArray(
+            servicesList.stream()
+                .map(
+                    service -> {
+                      Bundle map = new Bundle();
+                      map.putString("id", service.getId());
+                      map.putString(
+                          "capabilities",
+                          AccessibilityServiceInfo.capabilityToString(service.getCapabilities()));
```

Comment:
I don't think this will work, as the capabilities int is a bitmask of multiple capabilities set. You'll need to do similar work to what is done in `AccessibilityServiceInfo.appendCapabilities()` to pull out each capability individually.

https://cs.android.com/android/platform/superproject/+/master:frameworks/base/core/java/android/accessibilityservice/AccessibilityServiceInfo.java;l=1187?q=appendCapabilities